### ISSUE TITLE
No billing report generation option on Drupal 8 dev portal

### ISSUE THREAD

**Issue by vchikane** [apigee/apigee-m10n-drupal#259]
## Description
We are setting up monetization and configured the Monetization module on the Drupal 8 based dev portal. We have done some transactions and now I would like to see the billing report for the current month. But on Dev Portal, I only see to generate prepaid credit report or revenue report. There is no option to generate a billing report.

Could someone please confirm how to generate billing reports on the Dev Portal?

## Actual Behavior
What happened after the steps to reproduce.

## Screenshots
Attached is a screenshot that shows the revenue report option.
![Drupal8_Monetization_Report](https://user-images.githubusercontent.com/11797928/109115731-7c8dc480-7765-11eb-8772-d3cf8e03d36c.JPG)

## Version Info
This can be the version you can see on admin/modules in Drupal or 
the output of this command: `composer show`.  Add Drupal core and
other version information if needed.

**Comment by phdhiren** [comment]
@vchikane , Did you check [documentation](https://www.drupal.org/docs/contributed-modules/apigee-monetization) on drupal.org, Is that helpful?

**Comment by vchikane** [comment]
Hi @phdhiren, Yes I had setup monetization based on the documentation only, but I don't find anything related to reports.

**Comment by phdhiren** [comment]
@vchikane, After selecting proper date, it will download csv report for the user.

**Comment by vchikane** [comment]
@phdhiren Yes, I have downloaded the report and could see that it shows all the transactions done by that developer. 
The text on the reports page "Download revenue reports" is confusing, since, on Apigee monetization, we have both revenue report and billing report separately, hence I thought on the dev portal also, there should be two different types of reports.

Thanks for your suggestions.

**Comment by phdhiren** [comment]
@jbennett13,  @cnovak, How we can improve label on module / documentation?

**Comment by cnovak** [comment]
@phdhiren can you check to see what API is used to create these reports? That way we can validate what data is being shown.

**Comment by vchikane** [comment]
@phdhiren, I compared this with the billing report on Apigee edge. My requirement is still there that I want to download a summary billing-report. But there is no option to download the billing-report on the developer portal.

**Comment by phdhiren** [comment]
The current option is revenue detail under Reports menu. The billing report can be added as enhancement to the module
